Tests/schemas/test-playbook.yml

```diff
@@ -0,0 +1,211 @@
+type: map
```

Comment:
we might be able to avoid all this duplication

we can use the `pykwalify` validation with schema data (and not path)
![image](https://user-images.githubusercontent.com/10514677/44059767-1bfa05b4-9f5b-11e8-8c2f-1cef0977a934.png)

so we can read the playbook file, and change the relevant part instead duplicate this file